apps/desktop-wallet/package.json

```diff
@@ -59,7 +59,7 @@
     "@alephium/shared": "workspace:*",
     "@alephium/shared-crypto": "workspace:*",
     "@alephium/shared-react": "workspace:*",
-    "@alephium/token-list": "^0.0.12",
+    "@alephium/token-list": "^0.0.19",
```

Comment:
While working on fetching tokens I realized that the `TokenInfo` type that comes from this package was very out of date. So I updated the package and made the corresponding changes (see last comment)